### ISSUE TITLE
Can't create Ubuntu Focal Fossa VM with Multipass on Windows 10

### ISSUE THREAD

**Issue by pcgeek86** [canonical/multipass#1506]
**Describe the bug**
I am trying to create an Ubuntu Focal Fossa virtual machine on Windows 10 build 1909 with Hyper-V, using Multipass 1.2.

**To Reproduce**
How, and what happened?
1. `multipass find` doesn't return a Focal Fossa image

**Expected behavior**
What did you expect to happen?

`multipass find` returns a Focal Fossa image

**Logs**

```
PS C:\Users\TrevorSullivan> multipass find release:
Image                   Aliases           Version          Description
release:16.04           xenial            20200407         Ubuntu 16.04 LTS
release:18.04           bionic,lts        20200407         Ubuntu 18.04 LTS
PS C:\Users\TrevorSullivan> multipass find daily:
find failed: daily is not a supported remote. Please use `multipass find` for list of supported images.
PS C:\Users\TrevorSullivan>
PS C:\Users\TrevorSullivan> multipass version
multipass  1.2.0+win
multipassd 1.2.0+win
```

**Comment by Saviq** [comment]
Hi @pcgeek86!

Watch this space, it's coming asap :)

**Comment by Saviq** [comment]
@pcgeek86 here's a release candidate with those enabled - we just needed to make sure everything's in order :)

https://github.com/canonical/multipass/commit/c80b31f6dbd956562ee723cd45ad593e9171b0e9#commitcomment-38713603

**Comment by Saviq** [comment]
And released: [v1.2.1](https://github.com/canonical/multipass/releases/tag/v1.2.1).

**Comment by pcgeek86** [comment]
Thank you! I will be installing this shortly! 😁🍺
